src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -462,13 +462,6 @@ public static DetailAST parseWithComments(FileContents contents)
     }
 
     @Override
-    public void destroy() {
```

Comment:
> Check tests are failing

Before his latest push, there were no failing tests just code coverage.

> problem that super implementation is empty so it will do no effect on test.

Yes, so add a child to TreeWalker and verify the check's destroy is called.
With TreeWalker 'destroy' removed, check's destroy will never get called.